.azure/templates/jobs/build/push_containers.yaml

```diff
@@ -0,0 +1,64 @@
+jobs:
+  - job: 'push_containers'
+    displayName: 'Push Containers'
+    # Set timeout for jobs
+    timeoutInMinutes: 60
+    # Base system
+    pool:
+      vmImage: 'Ubuntu-20.04'
+    # Pipeline steps
+    steps:
+      # Install Prerequisites
+      - template: "../../steps/prerequisites/install_docker.yaml"
+      - template: "../../steps/prerequisites/install_yq.yaml"
+
+      # Get the container archives
+      - ${{ each arch in parameters.architectures }}:
+        - task: DownloadPipelineArtifact@2
+          inputs:
+            source: '${{ parameters.artifactSource }}'
+            artifact: Containers-${{ arch }}
+            path: $(System.DefaultWorkingDirectory)/
+            project: '${{ parameters.artifactProject }}'
+            pipeline: '${{ parameters.artifactPipeline }}'
+            runVersion: '${{ parameters.artifactRunVersion }}'
+            runId: '${{ parameters.artifactRunId }}'
+          displayName: "Download the ${{ arch }} container archives"
+        - bash: tar -xvf containers-${{ arch }}.tar
+          displayName: "Untar the ${{ arch }} containers"
+
+      # Push containers and their manifests
+      - bash: "docker login -u $DOCKER_USER -p $DOCKER_PASS $DOCKER_REGISTRY"
+        displayName: "Login to container registry"
+        env:
+          BUILD_REASON: $(Build.Reason)
+          BRANCH: $(Build.SourceBranch)
+          DOCKER_USER: $(QUAY_USER)
+          DOCKER_PASS: $(QUAY_PASS)
+          DOCKER_REGISTRY: "quay.io"
+      - bash: "make docker_delete_manifest"
+        displayName: "Delete existing container manifests"
+        env:
+          BUILD_REASON: $(Build.Reason)
+          BRANCH: $(Build.SourceBranch)
+          DOCKER_REGISTRY: "quay.io"
+          DOCKER_ORG: "strimzi"
+          DOCKER_TAG: '${{ parameters.dockerTag }}'
```

Comment:
wouldn't be better to set those env vars for the whole job? If it's possible of course, I am not sure now, but I think it should be

Reply:
I'm not aware of any way to do that. If there is, I can do it. But it might be a bit confusing since some env-vars might be defined more centrally, others which to for example define secrets would be only locally for the steps which actually need them for example.

Reply:
ye that makes sense...it should be possible with `variables` key on same level as `steps` but keep it as it is